airbyte-cdk/python/airbyte_cdk/sources/declarative/declarative_component_schema.yaml

```diff
@@ -1383,6 +1369,13 @@ definitions:
         anyOf:
           - "$ref": "#/definitions/SessionTokenRequestApiKeyAuthenticator"
           - "$ref": "#/definitions/SessionTokenRequestBearerAuthenticator"
+      decoder:
```

Comment:
Can we really have an authentication request that returns JSONL or iterable format?

Reply:
No! You are correct. I've removed the JSONL and Iterable decoders from the examples.

Additionally, I've added an `isinstance` check to verify the selected model is the `JsonDecoder`, which matches the other checks within the paginator/pagination strategies.